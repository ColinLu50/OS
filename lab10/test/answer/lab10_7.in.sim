11420
1115
1576
888
1564
854
