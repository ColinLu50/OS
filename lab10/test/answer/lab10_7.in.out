FCFS
677 -> 219 -> 6 -> 477 -> 755 -> 275 -> 151 -> 379 -> 433 -> 288 -> 364 -> 111 -> 830 -> 622 -> 890 -> 197 -> 899 -> 394 -> 236 -> 32 -> 536 -> 368 -> 299 -> 18 -> 239 -> 362 -> 400 -> 148 -> 497 -> 292 -> 310 -> 171 -> 81 -> 716 -> 19 -> 114 -> 570 -> 231 -> 416 -> 735
11420
SSTF
677 -> 716 -> 735 -> 755 -> 830 -> 890 -> 899 -> 622 -> 570 -> 536 -> 497 -> 477 -> 433 -> 416 -> 400 -> 394 -> 379 -> 368 -> 364 -> 362 -> 310 -> 299 -> 292 -> 288 -> 275 -> 239 -> 236 -> 231 -> 219 -> 197 -> 171 -> 151 -> 148 -> 114 -> 111 -> 81 -> 32 -> 19 -> 18 -> 6
1115
SCAN
677 -> 622 -> 570 -> 536 -> 497 -> 477 -> 433 -> 416 -> 400 -> 394 -> 379 -> 368 -> 364 -> 362 -> 310 -> 299 -> 292 -> 288 -> 275 -> 239 -> 236 -> 231 -> 219 -> 197 -> 171 -> 151 -> 148 -> 114 -> 111 -> 81 -> 32 -> 19 -> 18 -> 6 -> 0 -> 716 -> 735 -> 755 -> 830 -> 890 -> 899
1576
C-SCAN
677 -> 622 -> 570 -> 536 -> 497 -> 477 -> 433 -> 416 -> 400 -> 394 -> 379 -> 368 -> 364 -> 362 -> 310 -> 299 -> 292 -> 288 -> 275 -> 239 -> 236 -> 231 -> 219 -> 197 -> 171 -> 151 -> 148 -> 114 -> 111 -> 81 -> 32 -> 19 -> 18 -> 6 -> 0 -> 927 -> 899 -> 890 -> 830 -> 755 -> 735 -> 716
888
LOOK
677 -> 622 -> 570 -> 536 -> 497 -> 477 -> 433 -> 416 -> 400 -> 394 -> 379 -> 368 -> 364 -> 362 -> 310 -> 299 -> 292 -> 288 -> 275 -> 239 -> 236 -> 231 -> 219 -> 197 -> 171 -> 151 -> 148 -> 114 -> 111 -> 81 -> 32 -> 19 -> 18 -> 6 -> 716 -> 735 -> 755 -> 830 -> 890 -> 899
1564
C-LOOK
677 -> 622 -> 570 -> 536 -> 497 -> 477 -> 433 -> 416 -> 400 -> 394 -> 379 -> 368 -> 364 -> 362 -> 310 -> 299 -> 292 -> 288 -> 275 -> 239 -> 236 -> 231 -> 219 -> 197 -> 171 -> 151 -> 148 -> 114 -> 111 -> 81 -> 32 -> 19 -> 18 -> 6 -> 899 -> 890 -> 830 -> 755 -> 735 -> 716
854
